assets/dev/scss/frontend/widgets/spacer.scss

```diff
@@ -1,10 +1,35 @@
-.elementor-spacer-inner {
-	.e-container.e-container--row & {
-		width: var( --spacer-size );
```

Comment:
I have redone this widget completely because the original .e-container.e-container--row/column classes are not responsive.